paywall/src/constants.js

```diff
@@ -44,8 +44,8 @@ export const ACCOUNT_REGEXP = new RegExp(accountRegex)
 
 // helpers for the LOCK_PATH_NAME_REGEXP
 const prefix = '[a-z0-9]+'
-const urlEncodedRedirectUrl = '[^#?]+'
-const userAccount = accountRegex
+const urlEncodedUrl = '[^#?]+'
```

Comment:
Consistency alert! Use upper case for CONSTANTS!
Also maybe something that says it's a regex: URL_ENCODED_REGEX?

Reply:
per our conversation, as these are private, I'm going to keep them lower-cased, but add "private" to the comment above them to help clarify that.